guides/common/modules/proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc

```diff
@@ -15,9 +18,12 @@ However, {Project} does not remove the back-end services on the operating system
 --foreman-proxy-tftp false
 ----
 
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets* and select a subnet.
+. For every subnet where the {SmartProxy} is set as a DHCP, DNS, or TFTP proxy, disable the proxies:
```

Comment:
```suggestion
. Disable the {SmartProxy} integration for every subnet:
```

just a suggestion; mostly because I would like to avoid the word "proxies". The features are also listed in line 26.